release/src/Util.hs

```diff
@@ -292,7 +294,7 @@ artifactFiles artifact@Artifact{..} = do
         -- ^ Note that the Scaladoc is specified with the "javadoc" classifier.
 
 mainArtifactPath :: E.MonadThrow m => Text -> Artifact a -> m (Path Rel File)
-mainArtifactPath name artifact = parseRelFile $ unpack $ mainFileName (artReleaseType artifact) name
+mainArtifactPath name artifact = maybe (parseRelFile $ unpack $ mainFileName (artReleaseType artifact) name) return (artMainJar artifact)
```

Comment:
Why do we need this? The existing logic for `lib-jar` should turn `ledger_api_proto_proto_jar` into `libledger_api_proto_proto_jar.jar` which seems to be exactly what you’re setting it to via `main-jar`?

Reply:
Ugh, I added this because I was using `jar` and then discovered that doesn't work at all because we _need_ Javadoc and sources (even if empty). So I made that work, and now this is unnecessary as you say. I'll revert it.